service/matching/db.go

```diff
@@ -49,11 +50,12 @@ const (
 type (
 	taskQueueDB struct {
 		sync.Mutex
-		queue    *PhysicalTaskQueueKey
-		rangeID  int64
-		ackLevel int64
-		store    persistence.TaskManager
-		logger   log.Logger
+		queue        *PhysicalTaskQueueKey
+		rangeID      int64
+		ackLevel     int64
+		store        persistence.TaskManager
+		maxReadLevel int64
```

Comment:
use https://pkg.go.dev/sync/atomic#Int64 instead of a plain value, it's easier to work with and prevents bugs

Reply:
acknowledged